src/main/java/nextstep/courses/infrastructure/JdbcSessionCoverImageRepository.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.image.SessionCoverImage;
+import nextstep.courses.domain.port.SessionCoverImageRepository;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.util.Objects;
+
+@Repository("sessionCoverImageRepository")
+public class JdbcSessionCoverImageRepository implements SessionCoverImageRepository {
+
+    private JdbcOperations jdbcTemplate;
+
+    public JdbcSessionCoverImageRepository(JdbcOperations jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public SessionCoverImage findById(Long id) {
+        String sql = "select id, session_id, volume, fileName, width, height, file_path from session_cover_image where id = ?";
+        RowMapper<SessionCoverImage> rowMapper = (rs, rowNum) -> new SessionCoverImage(
+                rs.getLong(1),
+                rs.getLong(2),
+                rs.getInt(3),
+                rs.getString(4),
+                rs.getInt(5),
+                rs.getInt(6),
+                rs.getString(7)
+        );
+        return jdbcTemplate.queryForObject(sql, rowMapper, id);
+    }
+
+    @Override
+    public Long save(SessionCoverImage sessionCoverImage) {
+        String sql = "insert into session_cover_image (session_id, fileName, volume ,extension, height, width, file_path) values(?,?,?,?,?,?,?)";
+
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(sql, new String[] {"id"});
+            ps.setLong(1, sessionCoverImage.getSessionId());
+            ps.setString(2, sessionCoverImage.getFileName());
+            ps.setInt(3, sessionCoverImage.getCoverImageVolume().getSize());
```

Comment:
```suggestion
            ps.setInt(3, sessionCoverImage.getCoverImageVolumeSize()); 
```

 이런식으로 dot(.)을 줄여봐도 좋을 것 같아요. 외부에서 A라는 객체에서 c~d까지 직접 찾아가는건 중간에 있는 b~c등의 엔티티에 대한 커플링을 높혀 유지보수성을 떨어트리기도하고 디미터 법칙도 위배할 수 있으니까요 ㅎㅎ

Reply:
넵 좋은거 같아요..!